src/types/ApiResponses.ts

```diff
@@ -122,4 +127,28 @@ export interface GetRequestsResponse {
 
 export interface GetRequestResponse {
     request: Request;
+}
+
+// NOTIFICATIONS
+
+export interface PushTicketData {
+    status : "error" | "ok",
```

Comment:
colon indicates type, "error" nor "ok" is a type

u can either make this a string or enum